drpc.go

```diff
@@ -19,6 +19,7 @@ var (
 )
 
 // Transport is an interface describing what is required for a drpc connection.
+// Transport is a supertype of net.Conn
```

Comment:
I'm actually surprised that this didn't just extend `io.ReadWriteCloser`

Reply:
IMHO it doesn't make it clearer and it's a common practice to avoid a deep type hierarchy.

Would you like me to change it anyways?

And @ifraixedes do you have any suggestion that would make this change acceptable?

Reply:
@Erikvv no, I don't.

I was waiting for @zeebo to intervene here.

Reply:
I think writing the comment as `"Any net.Conn can be used as a Transport."` would be clearer than any type theoretic description.

Reply:
Thanks, changed to Egons suggestion.